faster_whisper/transcribe.py

```diff
@@ -593,6 +1204,10 @@ def generate_segments(
                     # don't skip if the logprob is high enough, despite the no_speech_prob
                     should_skip = False
 
+                # Skip if the logprob is very low (below the threshold value),
+                # despite no_speech_prob being low (ex: Too ambiguous outputs)
+                if avg_logprob < options.log_prob_low_threshold:
```

Comment:
Yes, the log needs to be better and we can set the `log_prob_low_threshold ` to `None`  by default.
There is no transcribed text probably because it's all hallucinated. Can you share the audio file?

Reply:
Ah you can hack by set language='zh' for english audio

Reply:
In that case, I would say it should not output transcribed text (so it's a desired behavior) as the content is not in "zh".